src/services/articles/articles.service.js

```diff
@@ -0,0 +1,22 @@
+// Initializes the `articles` service on path `/articles`
+const createService = require('feathers-mongoose');
+const createModel = require('../../models/articles.model');
```

Comment:
Should we not create a variable for file locations like MODELS so we can just require(MODELS+'/articles.model');?

Reply:
It's not a bad idea, if you want to do it, open an issue for it.